scripts/run_tests.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##########################################################################
+
+# INSTRUCTIONS:
+#
+# Run this script from the oppia root folder:
+#   bash scripts/run_tests.sh
+#
+# It runs all the tests, in this order:
+# - Frontend Karma unit tests
```

Comment:
I prefer keeping this order, because the karma tests run really quickly and I think it's better to fail fast. In the other order, if the backend tests pass and the karma tests fail, then when you make the change to fix the karma tests you should technically run everything again.